app/src/main/java/de/westnordost/streetcomplete/quests/diet_type/AddGlutenFree.kt

```diff
@@ -0,0 +1,50 @@
+package de.westnordost.streetcomplete.quests.diet_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.osm.isPlaceOrDisusedPlace
+import de.westnordost.streetcomplete.osm.updateWithCheckDate
+
+
+class AddGlutenFree : OsmFilterQuestType<DietAvailabilityAnswer>() {
+
+    override val elementFilter = """
+        nodes, ways with
+        (
+          amenity ~ restaurant|cafe|fast_food|food_court and food != no
+          or amenity ~ pub|nightclub|biergarten|bar and food = yes
+          or shop ~ supermarket|convenience
+        )
+        and (
+          !diet:gluten_free
+          or diet:gluten_free != only and diet:gluten_free older today -4 years
+        )
+    """
+    override val changesetComment = "Specify whether places are gluten-free"
+    override val wikiLink = "Key:diet:gluten_free"
+    override val icon = R.drawable.ic_quest_glutenfree
+    override val isReplacePlaceEnabled = true
+    override val achievements = listOf(CITIZEN)
+    override val defaultDisabledMessage = R.string.default_disabled_msg_go_inside
+
+    override val hint = R.string.quest_dietType_explanation_glutenfree
+
+    override fun getTitle(tags: Map<String, String>) = R.string.quest_dietType_glutenfree_name_title2
```

Comment:
rename to `R.string.quest_dietType_glutenfree_name_title`?

Reply:
OK, renamed (I don't know why, but all the other diet quests had `title2` in name, so I just kept it like that)